interface/src/app/tribe_client/geneset-result-table.tpl.html

```diff
@@ -1,15 +1,18 @@
-<div>
-  <table class="table table-striped" name="searchResults">
-    <tr>
-      <th>Title</th>
-      <th>Species</th>
-      <th>Creator</th>
-    </tr>
+<table class="table table-striped">
+  <tr>
+    <th></th>
+    <th>Title</th>
+    <th>Species</th>
+    <th>Creator</th>
+    <th>Genes</th>
+  </tr>
 
-    <tr ng-repeat="geneset in genesets">
-      <td>{{ geneset.title }}</td>
-      <td>{{ geneset.organism.scientific_name }}</td>
-      <td>{{ geneset.creator.username }}</td>
-    </tr>
-  </table>
-</div>
+  <tr ng-repeat="geneset in $ctrl.genesets">
```

Comment:
I'm still playing catch-up with you on the switch to Angular `.component`s, so forgive the naïve question: I don't see `$ctrl` declared anywhere. Does that come in for "free" via `.component`?